GameOfLife/Model/Game.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Mechanics of the game
+    /// </summary>
+    public class Game
+    {
+        public int [,] gameField;
+
+        /// <summary>
+        /// Starts the game with first random generation
+        /// </summary>
+        /// <param name="column">The number of columns entered by the user</param>
+        /// <param name="row">The number of rows entered by the user</param>
+        public Game (int column, int row)
+        {
+            gameField = new int [column, row];
+            SetRandomFirstGeneration();
+        }
+
+        /// <summary>
+        /// Sets first random generation of live and dead cells in game field
+        /// </summary>
+        public void SetRandomFirstGeneration()
+        {
+            Random random = new Random();
+            for (int x = 0; x < gameField.GetLength(0); x++)
```

Comment:
It is better to use more descriptive iteration variables then x and y.
Preferably column for x and row for y.

Reply:
Changes have been made.